admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -7,132 +7,33 @@
-
-==== PodDisruptionBudget is supported, but not guaranteed!
-
-A xref:../../admin_guide/managing_pods.adoc#managing-pods-poddisruptionbudget[Disruption Budget (PDB)] allows application owners to limit the number Pods of a replicated application that are down simultaneously from voluntary disruptions. {product-title} supports PDB when preempting Pods, but respecting PDB is best effort. The Scheduler tries to find victims whose PDB are not violated by preemption, but if no such victims are found, preemption will still happen, and lower priority Pods will be removed despite their PDBs being violated.
-
-==== Inter-Pod affinity on lower-priority Pods
-
-A Node is considered for preemption only when the answer to this question is yes: “If all the Pods with lower priority than the pending Pod are removed from the Node, can the pending Pod be scheduled on the Node?”
-
-Preemption does not necessarily remove all lower-priority Pods. If the pending Pod can be scheduled by removing fewer than all lower-priority Pods, then only a portion of the lower-priority Pods are removed. Even so, the answer to the preceding question must be yes. If the answer is no, the Node is not considered for preemption.
-
-If a pending Pod has inter-pod affinity to one or more of the lower-priority Pods on the Node, the inter-Pod affinity rule cannot be satisfied in the absence of those lower-priority Pods. In this case, the scheduler does not preempt any Pods on the Node. Instead, it looks for another Node. The scheduler might find a suitable Node or it might not. There is no guarantee that the pending Pod can be scheduled.
-
-Our recommended solution for this problem is to create inter-Pod affinity only towards equal or higher priority Pods.
-
-==== Cross node preemption
-
-Suppose a Node N is being considered for preemption so that a pending Pod P can be scheduled on N. P might become feasible on N only if a Pod on another Node is preempted. Here’s an example:
-
-. Pod P is being considered for Node N.
-. Pod Q is running on another Node in the same Zone as Node N.
-. Pod P has Zone-wide anti-affinity with Pod Q (topologyKey: failure-domain.beta.kubernetes.io/zone).
-. There are no other cases of anti-affinity between Pod P and other Pods in the Zone.
-. In order to schedule Pod P on Node N, Pod Q can be preempted, but scheduler does not perform cross-node preemption. So, Pod P will be deemed unschedulable on Node N.
-
-If Pod Q were removed from its Node, the Pod anti-affinity violation would be gone, and Pod P could possibly be scheduled on Node N.
+//Disabling Priority and Preemption
```

Comment:
@kalexand-rh This would be a stand-alone assembly just for disabling? Or, that has only disabling?

Reply:
@mburke5678, I'd probably make it separate since it's a different use case. (It's your call, though - it might make more sense in 4.0 when we're trying to make more isolated assemblies.)